clients/cli/src/flops.rs

```diff
@@ -30,5 +30,5 @@ pub fn measure_flops() -> f32 {
         .sum::<f64>()
         / NUM_REPEATS as f64; // Average the FLOPS over all repeats
 
-     (avg_flops / 1e9) as f32 // Convert to GFLOP/s and cast to f32
+    (avg_flops / 1e9) as f32 // Convert to GFLOP/s and cast to f32
```

Comment:
not sure if you want to include these 2 unrelated files.

Reply:
It's an extra space that should be removed. It was caught running the formatter on my PR. I think the issue is that the formatter doesn't run automatically on external PRs.